CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - When running `check_contracts` on a class with type aliases as type annotations for its attributes, the `NameError`
   that appears (which indicates that the type alias is undefined) is now resolved.
 - The default value of `pyta-number-of-messages` is now 0. This automatically displays all occurrences of the same error.
+- For the contract checking `new_setattr` function, the check for `ENABLE_CONTRACT_CHECKING` is now at the top of the
```

Comment:
Update this to remove the reference to `ENABLE_CONTRACT_CHECKING`. Also you don't need to go into details about the efficiency improvements: reword to just say "improve efficiency" in the changelog entry.